Building on master fails with g++ 13.1.1

Building with g++ 13.1.1 fails due to missing includes in
```
include/SFML/Network/Packet.hpp
include/SFML/System/String.hpp
include/SFML/System/Utf.hpp
```
They all miss `<cstdint>`.
## Your environment

* Fedora 38
* git master
* g++ 13.1.1, `cmake  -DSFML_BUILD_TEST_SUITE=TRUE` + `cmake --build`
* Special compiler flags used: None

## Steps to reproduce
I upgraded from Fedora 37 (which used g++ 12) to 38.

## Build result:
```none
$ cmake --build .
[  0%] Building CXX object src/SFML/System/CMakeFiles/sfml-system.dir/Clock.cpp.o
[  1%] Building CXX object src/SFML/System/CMakeFiles/sfml-system.dir/Err.cpp.o
[  1%] Building CXX object src/SFML/System/CMakeFiles/sfml-system.dir/Sleep.cpp.o
[  1%] Building CXX object src/SFML/System/CMakeFiles/sfml-system.dir/String.cpp.o
In file included from /home/ted/git/SFML/include/SFML/System/String.hpp:32,
                 from /home/ted/git/SFML/src/SFML/System/String.cpp:28:
/home/ted/git/SFML/include/SFML/System/Utf.hpp:71:40: error: ‘std::uint32_t’ has not been declared
   71 |     static In decode(In begin, In end, std::uint32_t& output, std::uint32_t replacement = 0);
      |                                        ^~~
/home/ted/git/SFML/include/SFML/System/Utf.hpp:71:63: error: ‘std::uint32_t’ has not been declared
   71 |     static In decode(In begin, In end, std::uint32_t& output, std::uint32_t replacement = 0);
      |                                                               ^~~
/home/ted/git/SFML/include/SFML/System/Utf.hpp:87:23: error: ‘std::uint32_t’ has not been declared
   87 |     static Out encode(std::uint32_t input, Out output, std::uint8_t replacement = 0);
      |                       ^~~
/home/ted/git/SFML/include/SFML/System/Utf.hpp:87:56: error: ‘std::uint8_t’ has not been declared
   87 |     static Out encode(std::uint32_t input, Out output, std::uint8_t replacement = 0);
      |                                                        ^~~
/home/ted/git/SFML/include/SFML/System/Utf.hpp:277:40: error: ‘std::uint32_t’ has not been declared
  277 |     static In decode(In begin, In end, std::uint32_t& output, std::uint32_t replacement = 0);
      |                                        ^~~
/home/ted/git/SFML/include/SFML/System/Utf.hpp:277:63: error: ‘std::uint32_t’ has not been declared
  277 |     static In decode(In begin, In end, std::uint32_t& output, std::uint32_t replacement = 0);
      |                                                               ^~~
/home/ted/git/SFML/include/SFML/System/Utf.hpp:293:23: error: ‘std::uint32_t’ has not been declared
  293 |     static Out encode(std::uint32_t input, Out output, std::uint16_t replacement = 0);
      |                       ^~~
/home/ted/git/SFML/include/SFML/System/Utf.hpp:293:56: error: ‘std::uint16_t’ has not been declared
  293 |     static Out encode(std::uint32_t input, Out output, std::uint16_t replacement = 0);
      |                                                        ^~~
/home/ted/git/SFML/include/SFML/System/Utf.hpp:484:40: error: ‘std::uint32_t’ has not been declared
  484 |     static In decode(In begin, In end, std::uint32_t& output, std::uint32_t replacement = 0);
      |                                        ^~~
/home/ted/git/SFML/include/SFML/System/Utf.hpp:484:63: error: ‘std::uint32_t’ has not been declared
  484 |     static In decode(In begin, In end, std::uint32_t& output, std::uint32_t replacement = 0);
      |                                                               ^~~
/home/ted/git/SFML/include/SFML/System/Utf.hpp:501:23: error: ‘std::uint32_t’ has not been declared
  501 |     static Out encode(std::uint32_t input, Out output, std::uint32_t replacement = 0);
      |                       ^~~
/home/ted/git/SFML/include/SFML/System/Utf.hpp:501:56: error: ‘std::uint32_t’ has not been declared
  501 |     static Out encode(std::uint32_t input, Out output, std::uint32_t replacement = 0);
      |                                                        ^~~
/home/ted/git/SFML/include/SFML/System/Utf.hpp:680:17: error: ‘uint32_t’ in namespace ‘std’ does not name a type; did you mean ‘wint_t’?
  680 |     static std::uint32_t decodeAnsi(In input, const std::locale& locale = std::locale());
      |                 ^~~~~~~~
      |                 wint_t
/home/ted/git/SFML/include/SFML/System/Utf.hpp:695:17: error: ‘uint32_t’ in namespace ‘std’ does not name a type; did you mean ‘wint_t’?
  695 |     static std::uint32_t decodeWide(In input);
      |                 ^~~~~~~~
      |                 wint_t
/home/ted/git/SFML/include/SFML/System/Utf.hpp:713:27: error: ‘std::uint32_t’ has not been declared
  713 |     static Out encodeAnsi(std::uint32_t codepoint, Out output, char replacement = 0, const std::locale& locale = std::locale());
      |                           ^~~
/home/ted/git/SFML/include/SFML/System/Utf.hpp:730:27: error: ‘std::uint32_t’ has not been declared
  730 |     static Out encodeWide(std::uint32_t codepoint, Out output, wchar_t replacement = 0);
      |                           ^~~
In file included from /home/ted/git/SFML/include/SFML/System/Utf.hpp:733:
/home/ted/git/SFML/include/SFML/System/Utf.inl:48:37: error: ‘std::uint32_t’ has not been declared
   48 | In Utf<8>::decode(In begin, In end, std::uint32_t& output, std::uint32_t replacement)
      |                                     ^~~
/home/ted/git/SFML/include/SFML/System/Utf.inl:48:60: error: ‘std::uint32_t’ has not been declared
   48 | In Utf<8>::decode(In begin, In end, std::uint32_t& output, std::uint32_t replacement)
      |                                                            ^~~
/home/ted/git/SFML/include/SFML/System/Utf.inl: In static member function ‘static In sf::Utf<8>::decode(In, In, int&, int)’:
/home/ted/git/SFML/include/SFML/System/Utf.inl:64:27: error: ‘uint32_t’ in namespace ‘std’ does not name a type; did you mean ‘wint_t’?
   64 |     static constexpr std::uint32_t offsets[6] =
      |                           ^~~~~~~~
      |                           wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl:71:57: error: ‘uint8_t’ in namespace ‘std’ does not name a type; did you mean ‘wint_t’?
   71 |     const int trailingBytes = trailing[static_cast<std::uint8_t>(*begin)];
      |                                                         ^~~~~~~
      |                                                         wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl:79:48: error: ‘uint8_t’ in namespace ‘std’ does not name a type; did you mean ‘wint_t’?
   79 |             case 5: output += static_cast<std::uint8_t>(*begin++); output <<= 6; [[fallthrough]];
      |                                                ^~~~~~~
      |                                                wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl:80:48: error: ‘uint8_t’ in namespace ‘std’ does not name a type; did you mean ‘wint_t’?
   80 |             case 4: output += static_cast<std::uint8_t>(*begin++); output <<= 6; [[fallthrough]];
      |                                                ^~~~~~~
      |                                                wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl:81:48: error: ‘uint8_t’ in namespace ‘std’ does not name a type; did you mean ‘wint_t’?
   81 |             case 3: output += static_cast<std::uint8_t>(*begin++); output <<= 6; [[fallthrough]];
      |                                                ^~~~~~~
      |                                                wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl:82:48: error: ‘uint8_t’ in namespace ‘std’ does not name a type; did you mean ‘wint_t’?
   82 |             case 2: output += static_cast<std::uint8_t>(*begin++); output <<= 6; [[fallthrough]];
      |                                                ^~~~~~~
      |                                                wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl:83:48: error: ‘uint8_t’ in namespace ‘std’ does not name a type; did you mean ‘wint_t’?
   83 |             case 1: output += static_cast<std::uint8_t>(*begin++); output <<= 6; [[fallthrough]];
      |                                                ^~~~~~~
      |                                                wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl:84:48: error: ‘uint8_t’ in namespace ‘std’ does not name a type; did you mean ‘wint_t’?
   84 |             case 0: output += static_cast<std::uint8_t>(*begin++);
      |                                                ^~~~~~~
      |                                                wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl:88:19: error: ‘offsets’ was not declared in this scope; did you mean ‘offsetof’?
   88 |         output -= offsets[trailingBytes];
      |                   ^~~~~~~
      |                   offsetof
/home/ted/git/SFML/include/SFML/System/Utf.inl: At global scope:
/home/ted/git/SFML/include/SFML/System/Utf.inl:103:5: error: ‘Out sf::Utf<8>::encode’ is not a static data member of ‘class sf::Utf<8>’
  103 | Out Utf<8>::encode(std::uint32_t input, Out output, std::uint8_t replacement)
      |     ^~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl:103:25: error: template definition of non-template ‘Out sf::Utf<8>::encode’
  103 | Out Utf<8>::encode(std::uint32_t input, Out output, std::uint8_t replacement)
      |                         ^~~~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl:103:25: error: ‘uint32_t’ is not a member of ‘std’; did you mean ‘wint_t’?
  103 | Out Utf<8>::encode(std::uint32_t input, Out output, std::uint8_t replacement)
      |                         ^~~~~~~~
      |                         wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl:103:45: error: expected primary-expression before ‘output’
  103 | Out Utf<8>::encode(std::uint32_t input, Out output, std::uint8_t replacement)
      |                                             ^~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl:103:58: error: ‘uint8_t’ is not a member of ‘std’; did you mean ‘wint_t’?
  103 | Out Utf<8>::encode(std::uint32_t input, Out output, std::uint8_t replacement)
      |                                                          ^~~~~~~
      |                                                          wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl: In static member function ‘static In sf::Utf<8>::next(In, In)’:
/home/ted/git/SFML/include/SFML/System/Utf.inl:154:10: error: ‘uint32_t’ is not a member of ‘std’; did you mean ‘wint_t’?
  154 |     std::uint32_t codepoint;
      |          ^~~~~~~~
      |          wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl:155:31: error: ‘codepoint’ was not declared in this scope
  155 |     return decode(begin, end, codepoint);
      |                               ^~~~~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl: In static member function ‘static Out sf::Utf<8>::fromAnsi(In, In, Out, const std::locale&)’:
/home/ted/git/SFML/include/SFML/System/Utf.inl:180:20: error: ‘uint32_t’ in namespace ‘std’ does not name a type; did you mean ‘wint_t’?
  180 |         const std::uint32_t codepoint = Utf<32>::decodeAnsi(*begin++, locale);
      |                    ^~~~~~~~
      |                    wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl:181:48: error: ‘codepoint’ was not declared in this scope
  181 |         output                        = encode(codepoint, output);
      |                                                ^~~~~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl: In static member function ‘static Out sf::Utf<8>::fromWide(In, In, Out)’:
/home/ted/git/SFML/include/SFML/System/Utf.inl:194:14: error: ‘uint32_t’ is not a member of ‘std’; did you mean ‘wint_t’?
  194 |         std::uint32_t codepoint = Utf<32>::decodeWide(*begin++);
      |              ^~~~~~~~
      |              wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl:195:42: error: ‘codepoint’ was not declared in this scope
  195 |         output                  = encode(codepoint, output);
      |                                          ^~~~~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl: In static member function ‘static Out sf::Utf<8>::toAnsi(In, In, Out, char, const std::locale&)’:
/home/ted/git/SFML/include/SFML/System/Utf.inl:221:14: error: ‘uint32_t’ is not a member of ‘std’; did you mean ‘wint_t’?
  221 |         std::uint32_t codepoint;
      |              ^~~~~~~~
      |              wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl:222:37: error: ‘codepoint’ was not declared in this scope
  222 |         begin  = decode(begin, end, codepoint);
      |                                     ^~~~~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl: In static member function ‘static Out sf::Utf<8>::toWide(In, In, Out, wchar_t)’:
/home/ted/git/SFML/include/SFML/System/Utf.inl:236:14: error: ‘uint32_t’ is not a member of ‘std’; did you mean ‘wint_t’?
  236 |         std::uint32_t codepoint;
      |              ^~~~~~~~
      |              wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl:237:37: error: ‘codepoint’ was not declared in this scope
  237 |         begin  = decode(begin, end, codepoint);
      |                                     ^~~~~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl: In static member function ‘static Out sf::Utf<8>::toLatin1(In, In, Out, char)’:
/home/ted/git/SFML/include/SFML/System/Utf.inl:253:14: error: ‘uint32_t’ is not a member of ‘std’; did you mean ‘wint_t’?
  253 |         std::uint32_t codepoint;
      |              ^~~~~~~~
      |              wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl:254:40: error: ‘codepoint’ was not declared in this scope
  254 |         begin     = decode(begin, end, codepoint);
      |                                        ^~~~~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl: In static member function ‘static Out sf::Utf<8>::toUtf16(In, In, Out)’:
/home/ted/git/SFML/include/SFML/System/Utf.inl:276:14: error: ‘uint32_t’ is not a member of ‘std’; did you mean ‘wint_t’?
  276 |         std::uint32_t codepoint;
      |              ^~~~~~~~
      |              wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl:277:37: error: ‘codepoint’ was not declared in this scope
  277 |         begin  = decode(begin, end, codepoint);
      |                                     ^~~~~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl: In static member function ‘static Out sf::Utf<8>::toUtf32(In, In, Out)’:
/home/ted/git/SFML/include/SFML/System/Utf.inl:291:14: error: ‘uint32_t’ is not a member of ‘std’; did you mean ‘wint_t’?
  291 |         std::uint32_t codepoint;
      |              ^~~~~~~~
      |              wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl:292:40: error: ‘codepoint’ was not declared in this scope
  292 |         begin     = decode(begin, end, codepoint);
      |                                        ^~~~~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl: At global scope:
/home/ted/git/SFML/include/SFML/System/Utf.inl:302:38: error: ‘std::uint32_t’ has not been declared
  302 | In Utf<16>::decode(In begin, In end, std::uint32_t& output, std::uint32_t replacement)
      |                                      ^~~
/home/ted/git/SFML/include/SFML/System/Utf.inl:302:61: error: ‘std::uint32_t’ has not been declared
  302 | In Utf<16>::decode(In begin, In end, std::uint32_t& output, std::uint32_t replacement)
      |                                                             ^~~
/home/ted/git/SFML/include/SFML/System/Utf.inl: In static member function ‘static In sf::Utf<16>::decode(In, In, int&, int)’:
/home/ted/git/SFML/include/SFML/System/Utf.inl:304:16: error: ‘uint16_t’ in namespace ‘std’ does not name a type; did you mean ‘wint_t’?
  304 |     const std::uint16_t first = *begin++;
      |                ^~~~~~~~
      |                wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl:307:10: error: ‘first’ was not declared in this scope
  307 |     if ((first >= 0xD800) && (first <= 0xDBFF))
      |          ^~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl:311:24: error: ‘uint32_t’ in namespace ‘std’ does not name a type; did you mean ‘wint_t’?
  311 |             const std::uint32_t second = *begin++;
      |                        ^~~~~~~~
      |                        wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl:312:18: error: ‘second’ was not declared in this scope
  312 |             if ((second >= 0xDC00) && (second <= 0xDFFF))
      |                  ^~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl: At global scope:
/home/ted/git/SFML/include/SFML/System/Utf.inl:342:5: error: ‘Out sf::Utf<16>::encode’ is not a static data member of ‘class sf::Utf<16>’
  342 | Out Utf<16>::encode(std::uint32_t input, Out output, std::uint16_t replacement)
      |     ^~~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl:342:26: error: template definition of non-template ‘Out sf::Utf<16>::encode’
  342 | Out Utf<16>::encode(std::uint32_t input, Out output, std::uint16_t replacement)
      |                          ^~~~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl:342:26: error: ‘uint32_t’ is not a member of ‘std’; did you mean ‘wint_t’?
  342 | Out Utf<16>::encode(std::uint32_t input, Out output, std::uint16_t replacement)
      |                          ^~~~~~~~
      |                          wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl:342:46: error: expected primary-expression before ‘output’
  342 | Out Utf<16>::encode(std::uint32_t input, Out output, std::uint16_t replacement)
      |                                              ^~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl:342:59: error: ‘uint16_t’ is not a member of ‘std’; did you mean ‘wint_t’?
  342 | Out Utf<16>::encode(std::uint32_t input, Out output, std::uint16_t replacement)
      |                                                           ^~~~~~~~
      |                                                           wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl: In static member function ‘static In sf::Utf<16>::next(In, In)’:
/home/ted/git/SFML/include/SFML/System/Utf.inl:381:10: error: ‘uint32_t’ is not a member of ‘std’; did you mean ‘wint_t’?
  381 |     std::uint32_t codepoint;
      |          ^~~~~~~~
      |          wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl:382:31: error: ‘codepoint’ was not declared in this scope
  382 |     return decode(begin, end, codepoint);
      |                               ^~~~~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl: In static member function ‘static Out sf::Utf<16>::fromAnsi(In, In, Out, const std::locale&)’:
/home/ted/git/SFML/include/SFML/System/Utf.inl:407:14: error: ‘uint32_t’ is not a member of ‘std’; did you mean ‘wint_t’?
  407 |         std::uint32_t codepoint = Utf<32>::decodeAnsi(*begin++, locale);
      |              ^~~~~~~~
      |              wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl:408:42: error: ‘codepoint’ was not declared in this scope
  408 |         output                  = encode(codepoint, output);
      |                                          ^~~~~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl: In static member function ‘static Out sf::Utf<16>::fromWide(In, In, Out)’:
/home/ted/git/SFML/include/SFML/System/Utf.inl:421:14: error: ‘uint32_t’ is not a member of ‘std’; did you mean ‘wint_t’?
  421 |         std::uint32_t codepoint = Utf<32>::decodeWide(*begin++);
      |              ^~~~~~~~
      |              wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl:422:42: error: ‘codepoint’ was not declared in this scope
  422 |         output                  = encode(codepoint, output);
      |                                          ^~~~~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl: In static member function ‘static Out sf::Utf<16>::toAnsi(In, In, Out, char, const std::locale&)’:
/home/ted/git/SFML/include/SFML/System/Utf.inl:445:14: error: ‘uint32_t’ is not a member of ‘std’; did you mean ‘wint_t’?
  445 |         std::uint32_t codepoint;
      |              ^~~~~~~~
      |              wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl:446:37: error: ‘codepoint’ was not declared in this scope
  446 |         begin  = decode(begin, end, codepoint);
      |                                     ^~~~~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl: In static member function ‘static Out sf::Utf<16>::toWide(In, In, Out, wchar_t)’:
/home/ted/git/SFML/include/SFML/System/Utf.inl:460:14: error: ‘uint32_t’ is not a member of ‘std’; did you mean ‘wint_t’?
  460 |         std::uint32_t codepoint;
      |              ^~~~~~~~
      |              wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl:461:37: error: ‘codepoint’ was not declared in this scope
  461 |         begin  = decode(begin, end, codepoint);
      |                                     ^~~~~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl: In static member function ‘static Out sf::Utf<16>::toUtf8(In, In, Out)’:
/home/ted/git/SFML/include/SFML/System/Utf.inl:491:14: error: ‘uint32_t’ is not a member of ‘std’; did you mean ‘wint_t’?
  491 |         std::uint32_t codepoint;
      |              ^~~~~~~~
      |              wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl:492:37: error: ‘codepoint’ was not declared in this scope
  492 |         begin  = decode(begin, end, codepoint);
      |                                     ^~~~~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl: In static member function ‘static Out sf::Utf<16>::toUtf32(In, In, Out)’:
/home/ted/git/SFML/include/SFML/System/Utf.inl:514:14: error: ‘uint32_t’ is not a member of ‘std’; did you mean ‘wint_t’?
  514 |         std::uint32_t codepoint;
      |              ^~~~~~~~
      |              wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl:515:40: error: ‘codepoint’ was not declared in this scope
  515 |         begin     = decode(begin, end, codepoint);
      |                                        ^~~~~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl: At global scope:
/home/ted/git/SFML/include/SFML/System/Utf.inl:525:42: error: ‘std::uint32_t’ has not been declared
  525 | In Utf<32>::decode(In begin, In /*end*/, std::uint32_t& output, std::uint32_t /*replacement*/)
      |                                          ^~~
/home/ted/git/SFML/include/SFML/System/Utf.inl:525:65: error: ‘std::uint32_t’ has not been declared
  525 | In Utf<32>::decode(In begin, In /*end*/, std::uint32_t& output, std::uint32_t /*replacement*/)
      |                                                                 ^~~
/home/ted/git/SFML/include/SFML/System/Utf.inl:534:5: error: ‘Out sf::Utf<32>::encode’ is not a static data member of ‘class sf::Utf<32>’
  534 | Out Utf<32>::encode(std::uint32_t input, Out output, std::uint32_t /*replacement*/)
      |     ^~~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl:534:26: error: template definition of non-template ‘Out sf::Utf<32>::encode’
  534 | Out Utf<32>::encode(std::uint32_t input, Out output, std::uint32_t /*replacement*/)
      |                          ^~~~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl:534:26: error: ‘uint32_t’ is not a member of ‘std’; did you mean ‘wint_t’?
  534 | Out Utf<32>::encode(std::uint32_t input, Out output, std::uint32_t /*replacement*/)
      |                          ^~~~~~~~
      |                          wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl:534:46: error: expected primary-expression before ‘output’
  534 | Out Utf<32>::encode(std::uint32_t input, Out output, std::uint32_t /*replacement*/)
      |                                              ^~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl:534:59: error: ‘uint32_t’ is not a member of ‘std’; did you mean ‘wint_t’?
  534 | Out Utf<32>::encode(std::uint32_t input, Out output, std::uint32_t /*replacement*/)
      |                                                           ^~~~~~~~
      |                                                           wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl:658:6: error: ‘uint32_t’ in namespace ‘std’ does not name a type; did you mean ‘wint_t’?
  658 | std::uint32_t Utf<32>::decodeAnsi(In input, [[maybe_unused]] const std::locale& locale)
      |      ^~~~~~~~
      |      wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl:670:6: error: ‘uint32_t’ in namespace ‘std’ does not name a type; did you mean ‘wint_t’?
  670 | std::uint32_t Utf<32>::decodeWide(In input)
      |      ^~~~~~~~
      |      wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl:684:5: error: ‘Out sf::Utf<32>::encodeAnsi’ is not a static data member of ‘class sf::Utf<32>’
  684 | Out Utf<32>::encodeAnsi(std::uint32_t codepoint, Out output, char replacement, [[maybe_unused]] const std::locale& locale)
      |     ^~~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl:684:30: error: template definition of non-template ‘Out sf::Utf<32>::encodeAnsi’
  684 | Out Utf<32>::encodeAnsi(std::uint32_t codepoint, Out output, char replacement, [[maybe_unused]] const std::locale& locale)
      |                              ^~~~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl:684:30: error: ‘uint32_t’ is not a member of ‘std’; did you mean ‘wint_t’?
  684 | Out Utf<32>::encodeAnsi(std::uint32_t codepoint, Out output, char replacement, [[maybe_unused]] const std::locale& locale)
      |                              ^~~~~~~~
      |                              wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl:684:54: error: expected primary-expression before ‘output’
  684 | Out Utf<32>::encodeAnsi(std::uint32_t codepoint, Out output, char replacement, [[maybe_unused]] const std::locale& locale)
      |                                                      ^~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl:684:62: error: expected primary-expression before ‘char’
  684 | Out Utf<32>::encodeAnsi(std::uint32_t codepoint, Out output, char replacement, [[maybe_unused]] const std::locale& locale)
      |                                                              ^~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl:684:81: error: expected identifier before ‘[’ token
  684 | Out Utf<32>::encodeAnsi(std::uint32_t codepoint, Out output, char replacement, [[maybe_unused]] const std::locale& locale)
      |                                                                                 ^
/home/ted/git/SFML/include/SFML/System/Utf.inl:684:97: error: type-specifier invalid in lambda
  684 | Out Utf<32>::encodeAnsi(std::uint32_t codepoint, Out output, char replacement, [[maybe_unused]] const std::locale& locale)
      |                                                                                                 ^~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl:684:103: error: type-specifier invalid in lambda
  684 | Out Utf<32>::encodeAnsi(std::uint32_t codepoint, Out output, char replacement, [[maybe_unused]] const std::locale& locale)
      |                                                                                                       ^~~
/home/ted/git/SFML/include/SFML/System/Utf.inl:684:97: error: parameter declaration before lambda declaration specifiers only optional with ‘-std=c++2b’ or ‘-std=gnu++2b’ [-Werror=c++23-extensions]
  684 | Out Utf<32>::encodeAnsi(std::uint32_t codepoint, Out output, char replacement, [[maybe_unused]] const std::locale& locale)
      |                                                                                                 ^~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl: In lambda function:
/home/ted/git/SFML/include/SFML/System/Utf.inl:684:114: error: expected ‘{’ before ‘&’ token
  684 | Out Utf<32>::encodeAnsi(std::uint32_t codepoint, Out output, char replacement, [[maybe_unused]] const std::locale& locale)
      |                                                                                                                  ^
/home/ted/git/SFML/include/SFML/System/Utf.inl: At global scope:
/home/ted/git/SFML/include/SFML/System/Utf.inl:684:116: error: ‘locale’ was not declared in this scope; did you mean ‘std::locale’?
  684 | Out Utf<32>::encodeAnsi(std::uint32_t codepoint, Out output, char replacement, [[maybe_unused]] const std::locale& locale)
      |                                                                                                                    ^~~~~~
      |                                                                                                                    std::locale
In file included from /usr/include/c++/13/locale:41,
                 from /home/ted/git/SFML/include/SFML/System/Utf.hpp:32:
/usr/include/c++/13/bits/locale_classes.h:62:9: note: ‘std::locale’ declared here
   62 |   class locale
      |         ^~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl:698:5: error: ‘Out sf::Utf<32>::encodeWide’ is not a static data member of ‘class sf::Utf<32>’
  698 | Out Utf<32>::encodeWide(std::uint32_t codepoint, Out output, wchar_t replacement)
      |     ^~~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl:698:30: error: template definition of non-template ‘Out sf::Utf<32>::encodeWide’
  698 | Out Utf<32>::encodeWide(std::uint32_t codepoint, Out output, wchar_t replacement)
      |                              ^~~~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl:698:30: error: ‘uint32_t’ is not a member of ‘std’; did you mean ‘wint_t’?
  698 | Out Utf<32>::encodeWide(std::uint32_t codepoint, Out output, wchar_t replacement)
      |                              ^~~~~~~~
      |                              wint_t
/home/ted/git/SFML/include/SFML/System/Utf.inl:698:54: error: expected primary-expression before ‘output’
  698 | Out Utf<32>::encodeWide(std::uint32_t codepoint, Out output, wchar_t replacement)
      |                                                      ^~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl:698:62: error: expected primary-expression before ‘wchar_t’
  698 | Out Utf<32>::encodeWide(std::uint32_t codepoint, Out output, wchar_t replacement)
      |                                                              ^~~~~~~
/home/ted/git/SFML/include/SFML/System/String.hpp:270:28: error: ‘uint8_t’ is not a member of ‘std’; did you mean ‘wint_t’?
  270 |     std::basic_string<std::uint8_t> toUtf8() const;
      |                            ^~~~~~~
      |                            wint_t
/home/ted/git/SFML/include/SFML/System/String.hpp:270:35: error: template argument 1 is invalid
  270 |     std::basic_string<std::uint8_t> toUtf8() const;
      |                                   ^
/home/ted/git/SFML/include/SFML/System/String.hpp:270:35: error: template argument 2 is invalid
/home/ted/git/SFML/include/SFML/System/String.hpp:270:35: error: template argument 3 is invalid
/home/ted/git/SFML/src/SFML/System/String.cpp: In constructor ‘sf::String::String(char, const std::locale&)’:
/home/ted/git/SFML/src/SFML/System/String.cpp:50:24: error: ‘decodeAnsi’ is not a member of ‘sf::Utf32’ {aka ‘sf::Utf<32>’}
   50 |     m_string += Utf32::decodeAnsi(ansiChar, locale);
      |                        ^~~~~~~~~~
/home/ted/git/SFML/src/SFML/System/String.cpp: In constructor ‘sf::String::String(wchar_t)’:
/home/ted/git/SFML/src/SFML/System/String.cpp:57:24: error: ‘decodeWide’ is not a member of ‘sf::Utf32’ {aka ‘sf::Utf<32>’}
   57 |     m_string += Utf32::decodeWide(wideChar);
      |                        ^~~~~~~~~~
/home/ted/git/SFML/src/SFML/System/String.cpp: At global scope:
/home/ted/git/SFML/src/SFML/System/String.cpp:171:24: error: ‘uint8_t’ is not a member of ‘std’; did you mean ‘wint_t’?
  171 | std::basic_string<std::uint8_t> String::toUtf8() const
      |                        ^~~~~~~
      |                        wint_t
/home/ted/git/SFML/src/SFML/System/String.cpp:171:31: error: template argument 1 is invalid
  171 | std::basic_string<std::uint8_t> String::toUtf8() const
      |                               ^
/home/ted/git/SFML/src/SFML/System/String.cpp:171:31: error: template argument 2 is invalid
/home/ted/git/SFML/src/SFML/System/String.cpp:171:31: error: template argument 3 is invalid
/home/ted/git/SFML/src/SFML/System/String.cpp: In member function ‘int sf::String::toUtf8() const’:
/home/ted/git/SFML/src/SFML/System/String.cpp:174:28: error: ‘uint8_t’ is not a member of ‘std’; did you mean ‘wint_t’?
  174 |     std::basic_string<std::uint8_t> output;
      |                            ^~~~~~~
      |                            wint_t
/home/ted/git/SFML/src/SFML/System/String.cpp:174:35: error: template argument 1 is invalid
  174 |     std::basic_string<std::uint8_t> output;
      |                                   ^
/home/ted/git/SFML/src/SFML/System/String.cpp:174:35: error: template argument 2 is invalid
/home/ted/git/SFML/src/SFML/System/String.cpp:174:35: error: template argument 3 is invalid
/home/ted/git/SFML/src/SFML/System/String.cpp:175:12: error: request for member ‘reserve’ in ‘output’, which is of non-class type ‘int’
  175 |     output.reserve(m_string.length());
      |            ^~~~~~~
In file included from /usr/include/c++/13/string:48,
                 from /usr/include/c++/13/bits/locale_classes.h:40:
/usr/include/c++/13/bits/stl_iterator.h: In instantiation of ‘class std::back_insert_iterator<int>’:
/home/ted/git/SFML/src/SFML/System/String.cpp:178:71:   required from here
/usr/include/c++/13/bits/stl_iterator.h:737:7: error: ‘int’ is not a class, struct, or union type
  737 |       operator=(const typename _Container::value_type& __value)
      |       ^~~~~~~~
/usr/include/c++/13/bits/stl_iterator.h:745:7: error: ‘int’ is not a class, struct, or union type
  745 |       operator=(typename _Container::value_type&& __value)
      |       ^~~~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl: In instantiation of ‘static Out sf::Utf<32>::fromAnsi(In, In, Out, const std::locale&) [with In = const char*; Out = std::back_insert_iterator<std::__cxx11::basic_string<char32_t> >]’:
/home/ted/git/SFML/src/SFML/System/String.cpp:77:28:   required from here
/home/ted/git/SFML/include/SFML/System/Utf.inl:562:31: error: ‘decodeAnsi’ was not declared in this scope; did you mean ‘encodeAnsi’?
  562 |         *output++ = decodeAnsi(*begin++, locale);
      |                     ~~~~~~~~~~^~~~~~~~~~~~~~~~~~
      |                     encodeAnsi
/home/ted/git/SFML/include/SFML/System/Utf.inl: In instantiation of ‘static Out sf::Utf<32>::fromAnsi(In, In, Out, const std::locale&) [with In = __gnu_cxx::__normal_iterator<const char*, std::__cxx11::basic_string<char> >; Out = std::back_insert_iterator<std::__cxx11::basic_string<char32_t> >]’:
/home/ted/git/SFML/src/SFML/System/String.cpp:87:20:   required from here
/home/ted/git/SFML/include/SFML/System/Utf.inl:562:31: error: ‘decodeAnsi’ was not declared in this scope; did you mean ‘encodeAnsi’?
  562 |         *output++ = decodeAnsi(*begin++, locale);
      |                     ~~~~~~~~~~^~~~~~~~~~~~~~~~~~
      |                     encodeAnsi
/home/ted/git/SFML/include/SFML/System/Utf.inl: In instantiation of ‘static Out sf::Utf<32>::fromWide(In, In, Out) [with In = const wchar_t*; Out = std::back_insert_iterator<std::__cxx11::basic_string<char32_t> >]’:
/home/ted/git/SFML/src/SFML/System/String.cpp:100:28:   required from here
/home/ted/git/SFML/include/SFML/System/Utf.inl:573:31: error: ‘decodeWide’ was not declared in this scope; did you mean ‘encodeWide’?
  573 |         *output++ = decodeWide(*begin++);
      |                     ~~~~~~~~~~^~~~~~~~~~
      |                     encodeWide
/home/ted/git/SFML/include/SFML/System/Utf.inl: In instantiation of ‘static Out sf::Utf<32>::fromWide(In, In, Out) [with In = __gnu_cxx::__normal_iterator<const wchar_t*, std::__cxx11::basic_string<wchar_t> >; Out = std::back_insert_iterator<std::__cxx11::basic_string<char32_t> >]’:
/home/ted/git/SFML/src/SFML/System/String.cpp:110:20:   required from here
/home/ted/git/SFML/include/SFML/System/Utf.inl:573:31: error: ‘decodeWide’ was not declared in this scope; did you mean ‘encodeWide’?
  573 |         *output++ = decodeWide(*begin++);
      |                     ~~~~~~~~~~^~~~~~~~~~
      |                     encodeWide
/home/ted/git/SFML/include/SFML/System/Utf.inl: In instantiation of ‘static Out sf::Utf<32>::toAnsi(In, In, Out, char, const std::locale&) [with In = __gnu_cxx::__normal_iterator<const char32_t*, std::__cxx11::basic_string<char32_t> >; Out = std::back_insert_iterator<std::__cxx11::basic_string<char> >]’:
/home/ted/git/SFML/src/SFML/System/String.cpp:150:18:   required from here
/home/ted/git/SFML/include/SFML/System/Utf.inl:594:28: error: conversion to ‘int’ from ‘char32_t’ may change the sign of the result [-Werror=sign-conversion]
  594 |         output = encodeAnsi(*begin++, output, replacement, locale);
      |                  ~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl: In instantiation of ‘static Out sf::Utf<32>::toWide(In, In, Out, wchar_t) [with In = __gnu_cxx::__normal_iterator<const char32_t*, std::__cxx11::basic_string<char32_t> >; Out = std::back_insert_iterator<std::__cxx11::basic_string<wchar_t> >]’:
/home/ted/git/SFML/src/SFML/System/String.cpp:164:18:   required from here
/home/ted/git/SFML/include/SFML/System/Utf.inl:605:28: error: conversion to ‘int’ from ‘char32_t’ may change the sign of the result [-Werror=sign-conversion]
  605 |         output = encodeWide(*begin++, output, replacement);
      |                  ~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl: In instantiation of ‘static Out sf::Utf<32>::toUtf8(In, In, Out) [with In = __gnu_cxx::__normal_iterator<const char32_t*, std::__cxx11::basic_string<char32_t> >; Out = std::back_insert_iterator<int>]’:
/home/ted/git/SFML/src/SFML/System/String.cpp:178:18:   required from here
/home/ted/git/SFML/include/SFML/System/Utf.inl:632:32: error: conversion to ‘int’ from ‘char32_t’ may change the sign of the result [-Werror=sign-conversion]
  632 |         output = Utf<8>::encode(*begin++, output);
      |                  ~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~
/home/ted/git/SFML/include/SFML/System/Utf.inl: In instantiation of ‘static Out sf::Utf<32>::toUtf16(In, In, Out) [with In = __gnu_cxx::__normal_iterator<const char32_t*, std::__cxx11::basic_string<char32_t> >; Out = std::back_insert_iterator<std::__cxx11::basic_string<char16_t> >]’:
/home/ted/git/SFML/src/SFML/System/String.cpp:192:19:   required from here
/home/ted/git/SFML/include/SFML/System/Utf.inl:642:33: error: conversion to ‘int’ from ‘char32_t’ may change the sign of the result [-Werror=sign-conversion]
  642 |         output = Utf<16>::encode(*begin++, output);
      |                  ~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~
cc1plus: all warnings being treated as errors
gmake[2]: *** [src/SFML/System/CMakeFiles/sfml-system.dir/build.make:118: src/SFML/System/CMakeFiles/sfml-system.dir/String.cpp.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:320: src/SFML/System/CMakeFiles/sfml-system.dir/all] Error 2
gmake: *** [Makefile:166: all] Error 2
```

--- a/include/SFML/Network/Packet.hpp
+++ b/include/SFML/Network/Packet.hpp
@@ -33,6 +33,7 @@
 #include <vector>
 
 #include <cstddef>
+#include <cstdint>
 
 
 namespace sf

--- a/include/SFML/System/String.hpp
+++ b/include/SFML/System/String.hpp
@@ -34,6 +34,8 @@
 #include <locale>
 #include <string>
 
+#include <cstdint>
+
 
 namespace sf
 {

--- a/include/SFML/System/Utf.hpp
+++ b/include/SFML/System/Utf.hpp
@@ -31,6 +31,7 @@
 
 #include <locale>
 
+#include <cstdint>
 #include <cstdlib>
 
 

